front/helpdesk.faq.php

```diff
@@ -43,7 +43,7 @@
 if (Session::getLoginUserID()) {
    Html::helpHeader(__('FAQ'), $_SERVER['PHP_SELF'], $_SESSION["glpiname"]);
 } else {
-   $_SESSION["glpilanguage"] = $CFG_GLPI['language'];
+   $_SESSION["glpilanguage"] = $_SESSION['glpifaqlanguage'] ?? $CFG_GLPI['language'];
```

Comment:
No need to create a new session variable.
```suggestion
   $_SESSION["glpilanguage"] = $_SESSION["glpilanguage"] ?? $CFG_GLPI['language'];
```